spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -24,10 +24,6 @@ class Availability < TunesBase
       # @return (Bool) b2b available for distribution
       attr_accessor :b2b_unavailable
 
-      # @return (Array of Spaceship::Tunes::B2bUser objects) A list of users set by user - if not
-      # then the b2b user list that is currently set
-      attr_accessor :b2b_users
```

Comment:
Deleting this `attr_accessor` still feels weird if we are still using `@b2b_users` 🤔 It feels like we should keep this since we are still technically using it but keep the `b2b_users` method changes like you have below

Reply:
well we define our own `b2b_users` method - i can also define a `b2b_users=` method as well. Let me see if i can keep it around though.